lib_eio/unix/fork_action.c

```diff
@@ -5,10 +5,14 @@
 #include <errno.h>
 
 #include <caml/mlvalues.h>
+#include <caml/unixsupport.h>
 
 #include "fork_action.h"
 
 void eio_unix_run_fork_actions(int errors, value v_actions) {
+  #ifdef _WIN32
```

Comment:
You can probably #ifdef out the whole function. It isn't called from OCaml code.